docs/stories/hooks/useShortcutEffect.stories.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+/* eslint-disable import/no-extraneous-dependencies */
+import { storiesOf } from '@storybook/react';
+import { Button } from '@buffetjs/core';
+import { useShortcutEffect } from '@buffetjs/hooks';
+import Presentation from '../ui/Presentation';
+import Pre from '../ui/Pre';
+
+const Story = () => {
+  const [enableHook, setEnableHook] = useState(false);
+
+  useShortcutEffect(
+    'tab',
+    () => {
+      alert('You pressed a key');
+    },
+    enableHook
+  );
+
+  return (
+    <Presentation
+      title="useEventListener"
+      description="Hook to quickly add shortcut effect."
+    >
+      <div style={{ marginBottom: 23 }}>
+        <Button
+          primary
+          type="button"
+          onClick={() => setEnableHook(prev => !prev)}
+        >
+          {enableHook ? 'Disable' : 'Enable'} hook
+        </Button>
+        {enableHook && (
+          <p style={{ fontSize: 13, marginTop: 13 }}>
+            Press tab to trigger the event
+          </p>
+        )}
+        <p style={{ fontSize: 13, marginTop: 17 }}>
+          The supported keys are alt, arrowUp, arrowDown, f, enter and tab.
```

Comment:
All the keys are triggered, is it what we want?

Reply:
Good catch! It is because I was using an alert which was blocking the hook. Changing it to `console.log` fixes the issue.

Reply:
Good catch! It is because I was using an alert which was blocking the hook. Changing it to `console.log` fixes the issue.